### ISSUE TITLE
Expatriated Belgian municipality

### ISSUE THREAD

**Issue by missinglink** [whosonfirst-data/whosonfirst-data#2212]
It seems that [Honnelles](https://spelunker.whosonfirst.org/id/1360697009) is part of Belgium, not France 🤔 

I wasn't able to find any reference of a French locality by that name, I also checked the location of the geometry.

It's not far away from where it should be 🤷‍♂️ 

- https://spelunker.whosonfirst.org/id/1360697009
- https://spelunker.whosonfirst.org/id/1125344213

References:

- https://www.openstreetmap.org/relation/1919117
- https://en.wikipedia.org/wiki/Honnelles

**Comment by missinglink** [comment]
I believe the geometry should be: `lat: 50.3520890, lng: 3.7310190`

**Comment by nvkelso** [comment]
I agree that's probably a bad name for that feature. It's not as clear what to do about it.

I suspect the existing French locality polygon of `Honnelles` in [1360697009](https://spelunker.whosonfirst.org/id/1360697009) could be better known as [Bry](https://fr.wikipedia.org/wiki/Bry_%28Nord%29?uselang=en) – which is also the [localadmin](https://spelunker.whosonfirst.org/id/404394511) name there and we already have a [locality record](https://spelunker.whosonfirst.org/id/1126040385) of that name as a point location nearby. On some maps I do see a neighbourhood named `La Houlette` in that same area. We could mark it as funky or deprecate it while also moving the polygon part of the record to the point locality (and expand it to cover the area to the west where the point is). We shouldn't reallocate it to Belgium.

The Belgium localadmin in [1125344213](https://spelunker.whosonfirst.org/id/1125344213) is already called Honnelles, and the big locality there is [Roisin](https://spelunker.whosonfirst.org/id/1259814579). Both of those are points now.

**Comment by justinelliotmeyers** [comment]
current france wof localities here

**Comment by justinelliotmeyers** [comment]
![Uploading Screenshot 2024-09-18 133726.png…]()

**Comment by justinelliotmeyers** [comment]
can confirm no locality in Nord, France has a name of Honnelles.

**Comment by missinglink** [comment]
https://youtu.be/8ddlv47t1tQ
